Source/Shared/BPConfiguration.m

```diff
@@ -78,7 +78,7 @@ typedef NS_OPTIONS(NSUInteger, BPOptionType) {
     {'o', "output-dir", BP_MASTER | BP_SLAVE, NO, NO, required_argument, NULL, BP_VALUE | BP_PATH, "outputDirectory",
         "Directory where to put output log files (bluepill only)."},
     {'j', "test-time-estimates-json", BP_MASTER, NO, NO, required_argument, NULL, BP_VALUE | BP_PATH, "testTimeEstimatesJsonFile",
-        "Directory where Bluepill looks for input files with test execution time estimates (bluepill only)."},
+        "Path of the input file with test execution time estimates."},
```

Comment:
We should also change the help of `--clone-simulator` to say that this flag is _always on_ and can't be turned off.